Constructive Code Review/outline.txt

```diff
@@ -0,0 +1,205 @@
+- Splash screen
+    - PR screenshot: “Your code is bad and you are bad. Have a bad day.”
+- Importance
+    - Who's done code review before?
+        - Who's done it sync?
+            - We don't really do that, aside from maybe some pair
+              programming.
```

Comment:
Added to the Clarity section so it doesn't hold up the front of the talk. Actually, deleted the whole sync/async thing from here as well, because it just kills the pacing.